js-objects/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "js-objects",
+  "version": "1.0.0",
+  "main": "objects.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

add here command to run your tests, f.e 
```suggestion
    "objects": "node js-objects/objects.js"
```